include/nsysccr/cdc.h

```diff
@@ -136,13 +136,14 @@ typedef enum CCRCDCExt
 
 typedef enum CCRCDCDrhStateEnum
 {
-   CCR_CDC_SYS_DRH_STATE_UNK0   = 0x00,
-   CCR_CDC_SYS_DRH_STATE_UNK1   = 0x01,
-   CCR_CDC_SYS_DRH_STATE_UNK2   = 0x02,
-   CCR_CDC_SYS_DRH_STATE_UNK3   = 0x03,
-   CCR_CDC_SYS_DRH_STATE_ECO    = 0x04,
-   CCR_CDC_SYS_DRH_STATE_UNK7F  = 0x7F,
-   CCR_CDC_SYS_DRH_STATE_CAFE   = 0xFF,
+   CCR_CDC_SYS_DRH_STATE_NORADIO = 0x00,
+   CCR_CDC_SYS_DRH_STATE_WII     = 0x01,
+   CCR_CDC_SYS_DRH_STATE_UNK2    = 0x02,
+   CCR_CDC_SYS_DRH_STATE_NODRC   = 0x03,
+   CCR_CDC_SYS_DRH_STATE_ECO     = 0x04,
+   //! On latest firmware this state is not recognized
```

Comment:
This state is definitely recognized on the latest firmware, even if it fails to enter it.
There are several references to it in the set and get state code.

Reply:
The suggestion is for state 7F, not state 04.

Reply:
Yeah, I'm talking about state 7F, I just put the comment one line too high.